src/ui/widgets/detail/DetailWidgetElement.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Icon, DropdownKebab } from 'patternfly-react';
+import { LinkMenuItem } from 'frontend-common-components';
+import { ROUTE_PAGE_DETAIL, ROUTE_PAGE_CONFIG } from 'app-init/router';
+
+const DetailWidgetElement = ({ widgetInfo }) => {
+  const renderWidgetInfo = () => {
+    const { data } = widgetInfo;
+    if (data) {
+      return Object.keys(widgetInfo.data)
+        .sort((a, b) => (a > b ? 1 : -1))
+        .map(info => (
+          <tr key={info}>
+            <td className="DetailWidgetElement__td">{info}</td>
+            <td className="DetailWidgetElement__td">{data[info].pageFullPath}</td>
+            <td className="DetailWidgetElement__td text-center">
+              <Icon
+                type="fa"
+                name={data[info].publish ? 'check' : 'close'}
+              />
+            </td>
+            <td className="DetailWidgetElement__td text-center">{data[info].framePublish}</td>
+            <td className="DetailWidgetElement__td text-center">
+              <Icon
+                type="fa"
+                name={data[info].draft ? 'check' : 'close'}
```

Comment:
Are you sure data[info] is not undefined? Could you use `get(data, `${info}.draft`)` ?